### PR TITLE
docs: update readme for pre-conditions

### PR DESCRIPTION
**Pull request by smrz2001**
Updated README to mention AWS resources that need to exist prior to the application of this module.

Eventually, we need to start creating these resources from within the module so that it become low-touch for user less familiar with AWS and Terraform.

### CODE REVIEW

**Review by v-stickykeys — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -9,11 +9,22 @@ There is no root module available for use. Instead use one of the submodules lik
...
+  version = "3.0.0"
   # insert variables here
 }
 ```
 
+## Pre-conditions
+
+Make sure that the following AWS resources exist prior to the application of this Terraform module:
+- VPC with 2 subnets in 2 AZs, each subnet tagged with `Ceramic` and the name of your environment, e.g. `dev`.
+- S3 bucket for Ceramic/IPFS data store
```
Comment (v-stickykeys) [inline README.md:21]: We should show examples of how to create these in the examples section. Or better yet allow them to be optionally created in the module itself. Module should be more self serve

**Review by v-stickykeys — CHANGES_REQUESTED** [review]
Oops sorry, shouldn't have approved this so fast 😅

File: README.md (hunk 1 above)
Comment (smrz2001) [reply README.md:21]: Yes, absolutely, creating in the module itself needs to happen soon (as I noted in the PR description) but should be done in subsequent PRs.

This PR is just meant to document what isn't covered by the module yet so people don't get confused.

Comment (smrz2001) [reply README.md:21]: ...versus right now, where they have to figure out through trial-and-error what is needed beforehand. This PR was in response to issues faced by a community member trying to use the module.

I'm assuming such resources will currently need to be created from the console, not sure if giving examples of how to do it in TF is worth it if we're going to add support to the module anyway.

Comment (smrz2001) [reply README.md:21]: Added a [ticket](https://linear.app/3boxlabs/issue/PLAT-526/allow-optionally-creating-resources-assumed-to-be-present-in-partner) to the next cycle to do this.

**Review by v-stickykeys — APPROVED** [review]
👍

**Pull request merged**